std_msgs/Header header

float32 p1_x
float32 p1_y
float32 p1_z
float32 p2_x
float32 p2_y
float32 p2_z
float32 p3_x
float32 p3_y
float32 p3_z
float32 p4_x
float32 p4_y
float32 p4_z